frontend/app/api/auth/register/route.ts

```
@@ -0,0 +1,46 @@
+    // If backend set a cookie, forward it to the browser
+    // Forward all cookies to the client
+    for (const cookie of setCookieHeaders) {
+      nextResponse.headers.append('set-cookie', cookie)
+    }
+    
+    return nextResponse
+    console.error('Error message:', error instanceof Error ? error.message : String(error))
+    return NextResponse.json(
+      { error: 'Failed to register user', details: error instanceof Error ? error.message : String(error) },
+      { status: 500 }
+    )
```

Biome (2.1.2)

[error] This code will never be reached ...

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)